app/helpers/notifications_helper.rb

```diff
@@ -23,4 +23,12 @@ def any_active_filters?
   def filtered_params(override = {})
     filters.merge(override)
   end
+
+  def archive_selected_button(custom_class=nil)
+    action = params[:archive] ? 'unarchive' : 'archive'
+    button_tag(type: "button",
+               class: "archive_toggle #{action}_selected #{custom_class}") do
+      "#{action} selected".capitalize
```

Comment:
@michaelrauh Variable `action` is used in the line 30 to construct a class name `archive_selected` or `unarchive_selected`. So the question is: are HTML class names case-sensitive?